docs/ml-features.md

```diff
@@ -815,7 +815,7 @@ The rescaled value for a feature E is calculated as,
 `\begin{equation}
   Rescaled(e_i) = \frac{e_i - E_{min}}{E_{max} - E_{min}} * (max - min) + min
 \end{equation}`
-For the case `E_{max} == E_{min}`, `Rescaled(e_i) = 0.5 * (max + min)`
+For the case `$E_{max} == E_{min}$`, `$Rescaled(e_i) = 0.5 * (max + min)$`
```

Comment:
Thanks for the review!

> Nit: write = instead of == in the first equation?

Both are ok for me, but in the MinMaxScaler's scaladoc we're also using `==`, so we update both places or leave it as is.

> It looks like this is repeated in the scaladoc for MinMaxScaler. Can that be updated too?

I don't think we can format math formulas in scaladoc, e.g. from [MinMaxScaler doc](https://spark.apache.org/docs/2.0.0-preview/api/java/org/apache/spark/ml/feature/MinMaxScaler.html) it seems not formatted at all.